### PR TITLE
docs(airtable) move airflow section to airflow rather than dataset docs

### PR DESCRIPTION
**Pull request by Nkdiaz**
moving the airflow maintenance sections out of dataset docs to the GTFS schedule airflow docs

### CODE REVIEW

File: docs/datasets_and_tables/transitdatabase.md [hunk 1]
```diff
@@ -12,27 +12,6 @@ We have chosen to group and maintain the tables in the Database as follows:
 
 While `organizations` and `services` are central to many of the tables, we have chosen to maintain them as part of the Transit Services Tables cluster which will be referenced by the other two.
 
-## Airtable things
```
Comment (evansiroky) [inline docs/datasets_and_tables/transitdatabase.md:15]: This is definitely airtable-specific documentation and should stay in here.

File: docs/airflow/overview.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
 (airflow)=
-# Airflow
 
-Cal-ITP has an instance of airflow that runs various tasks. More later...
+# Airtable
+
+## Primary Keys
```
Comment (evansiroky) [inline docs/airflow/overview.md:5]: This airtable does not appear to be relevant to airflow documentation.

**Review by evansiroky — CHANGES_REQUESTED** [review]
Seems like there may be a mixup and some confusion around airtable vs airflow.

**Comment by charlie-costanzo** [comment]
> Seems like there may be a mixup and some confusion around airtable vs airflow.

Ah sorry for the confusion here. This PR can be closed out in favor of the already merged PR #1007

**Comment by evansiroky** [comment]
> > Seems like there may be a mixup and some confusion around airtable vs airflow.
> 
> Ah sorry for the confusion here @evansiroky. This PR can be closed out in favor of the already merged PR #1007 which accomplishes this

thanks for the clarification
